libmachine/provision/boot2docker.go

```diff
@@ -50,13 +50,13 @@ func (provisioner *Boot2DockerProvisioner) Service(name string, action pkgaction
 }
 
 func (provisioner *Boot2DockerProvisioner) upgradeIso() error {
-	log.Infof("Stopping machine to do the upgrade...")
-
 	switch provisioner.Driver.DriverName() {
 	case "vmwarefusion", "vmwarevsphere":
 		return errors.New("Upgrade functionality is currently not supported for these providers, as they use a custom ISO.")
 	}
 
+	log.Infof("Stopping machine to do the upgrade...")
```

Comment:
I know this wasn't from you but we can just do `log.Info(..)` here (no `Infof`).  Would you mind updating?